frontend/index.html

```diff
@@ -7,10 +7,136 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>BIG data Group 11</title>
 </head>
+<style>
+    table {
+        width: 80%;
+        border-collapse: collapse;
+    }
+    th, td {
+        border: 1px solid #000000;
+        padding: 8px;
+        text-align: left;
+    }
+    th {
+        background-color: #ffffff;
+    }
+    body{
+        background-color: #eeeeee;
+    }
+</style>
 <body>
+
+<p style="font-size: 22px;">Welcome to <span style="font-size: larger; font-weight: bold">BIG</span> Data Management and Security Group 8: Shamtagram</p>
```

Comment:
Shamtagram or Instasham

Reply:
after a quick thinking; while instasham is arguably better, Shamtagram is just hilariously bad that I keep we keep it